trompace/mutations/itemlist.py

```diff
@@ -0,0 +1,197 @@
+                                                                itemlistorder)
+
+    args = {
+        "identifier": identifier,
+        "contributor": contributor,
+        "name": name,
+        "description": description,
+        "itemListOrder": StringConstant(itemlistorder)
+    }
+
+    args = filter_none_args(args)
+
+    return format_mutation("UpdateItemList", args)
+
+
+def mutation_delete_itemlist(identifier: str):
+    """Returns a mutation for deleting an itemlist object based
+    on the identifier.
+
+    Arguments:
+        identifier: The unique identifier of the itemlist object.
+
+    Returns:
+        The string for the mutation for deleting the itemlist object
+        based on the identifier.
+    """
+
+    return format_mutation("DeleteItemList", {"identifier": identifier})
+
+
+LISTITEM_ARGS_DOCS = """name: The name of the ItemList object.
+        contributor: A person, an organization, or a service responsible
+        for contributing the ItemList to the web resource.
+        This can be either a name or a base URL.
+        description: The description of the ItemList object
+        """
+
+
+@docstring_interpolate("listitem_args", LISTITEM_ARGS_DOCS)
+def mutation_create_listitem(contributor: str, name: str = None,
+                             description: str = None):
+    """Returns a mutation for creating a listitem object.
+
+    Arguments:
+        {listitem_args}
+
+    Returns:
+        The string for the mutation for creating the itemlist.
+    """
+    args = {
+        "contributor": contributor,
+        "name": name,
+        "description": description,
+    }
+
+    args = filter_none_args(args)
+
+    return format_mutation("CreateListItem", args)
+
+
+@docstring_interpolate("listitem_args", LISTITEM_ARGS_DOCS)
+def mutation_update_listitem(identifier, contributor: str, name: str = None,
+                             description: str = None):
+    """Returns a mutation for updating a listitem_args object.
+
+    Arguments:
+        identifier: The identifier of the listitem_args in the CE to be
+        updated.
+        {listitem_args}
+
+    Returns:
+        The string for the mutation for updating the listitem_args.
+    """
+    args = {
+        "identifier": identifier,
+        "contributor": contributor,
+        "name": name,
+        "description": description,
+    }
+
+    args = filter_none_args(args)
+
+    return format_mutation("UpdateListItem", args)
+
+
+def mutation_delete_listitem(identifier: str):
+    """Returns a mutation for deleting a listitem object based on the
+    identifier.
+
+    Arguments:
+        identifier: The unique identifier of the listitem object.
+
+    Returns:
+        The string for the mutation for deleting the listitem object based
+        on the identifier.
+    """
+    return format_mutation("DeleteListItem", {"identifier": identifier})
+
+
+def mutation_add_itemlist_itemlist_element(identifier_1: str,
```

Comment:
In some places where these methods can be used for multiple different types, I used `identifier_1` and `identifier_2`. However, in the case where we always know what the type is, we can use a more descriptive name.
In the case of this method, the first argument is always an itemlist identifier, so lets call it `itemlist_id` instead.

Remember that an ItemListElement can be any ThingInterface, so the documentation should be updated to indicate that this is possible